x-pack/plugins/runtime_fields/public/components/runtime_field_form/runtime_field_form.tsx

```diff
@@ -18,22 +19,25 @@ import {
 import { useForm, Form, FormHook, UseField, TextField, CodeEditor } from '../../shared_imports';
 import { RuntimeField } from '../../types';
 import { RUNTIME_FIELD_OPTIONS } from '../../constants';
-
 import { schema } from './schema';
 
+import './runtime_field_form.scss';
```

Comment:
Instead of importing component-level SCSS files directly here, please create an `index.scss` and import it in your main application file. Commonly, the `index.scss` lives at `public/` but it can also be in your components folder, if necessary. Main point being, we want to avoid the importation of multiple component level files in multiple places.

Reply:
@sebelga You can see an example of what Ryan's describing in the Search Profiler plugin, which I recently refactored to follow this pattern (https://github.com/elastic/kibana/pull/82085).

Reply:
On second thought...

I'm thinking we might be able to avoid any SCSS overrides altogether in this PR. In looking at this locally, it strikes me that - since this is akin to a form input - we should be coloring it as such by default. 

Specifically, I'm inclined to change the default editor theme's background color to `$euiFormBackgroundColor` in `src/plugins/kibana_react/public/code_editor/editor_theme.ts` (from its current empty/white color). This means we'd have to look at other instances of this CodeEditor component, but I suspect it's only used in a few spots (and it may not result in any changes for those instances any way).

If there are no objections, then you can remove your scss file, leave it with a white background for now, and I'll start a separate PR that changes the theme.

cc:/ @poffdeluxe since he originally built this for Canvas (I can clean up the container bg color there)

Reply:
@ryankeairns Is this what you're thinking? https://github.com/elastic/eui/issues/499#issuecomment-720394514

Reply:
> Instead of importing component-level SCSS files directly here, please create an index.scss and import it in your main application file.

I am a bit surprised here as it feels like going backward. We used to do that, then the recommended way (I think @cchaos mentioned it) was to keep the .scss file close to the component and import it directly from the component file. As we do here
* https://github.com/elastic/kibana/blob/master/x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/processor_form/processor_output/processor_output.tsx#L23
* https://github.com/elastic/kibana/blob/master/x-pack/plugins/index_management/public/application/components/component_templates/component_template_selector/component_templates_selector.tsx#L25)

I like that approach as it makes clearer the connection of the component with its style and we'd probably get the CSS lazyloaded if the component is lazyloaded.   So now we need to go back and declare a top-level index.scss?

> I'm thinking we might be able to avoid any SCSS overrides altogether in this PR. In looking at this locally, it strikes me that - since this is akin to a form input - we should be coloring it as such by default.

That'd be awesome if we could theme this globally. I did not want the runtime field editor to ship with a white background and no border as it lacked a frame for the user to work with. And from @cjcenizal PR I see the issue is from 2018 so I did not want to hold my breath on it 😄 
But if you'll work on it I will remove the .scss override on this PR then. 👍

Reply:
Yes, the current spec for adding new SASS files is to import them directly into the matching JS component file. https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#sass-files